examples/linearSolvers_example.cpp

```diff
@@ -15,7 +15,7 @@
 
 #include <iostream>
 #include <gismo.h>
-
+#include <gsSolver/gsMinResQLP.h>
```

Comment:
please add the include in the file src/gismo.h instead.

Reply:
Done